requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 -e .
-flake8==3.2.1
-ipdb==0.10.1
-pytest==3.0.5
-pytest-cov==2.4.0
```

Comment:
Please leave it as it is, I added pyup (https://pyup.io/) bot to this repo we will have automatic PR with dependency update, see example here https://github.com/aio-libs/aiomonitor/pull/10 Other aio-libs do the same